cvat/apps/authentication/utils.py

```diff
@@ -5,25 +5,31 @@
 from web3.auto import w3
 from eth_account.messages import encode_defunct
 from django.contrib.auth.hashers import get_hasher
-
+from django.utils.translation import ugettext_lazy as _
+from rest_framework import exceptions
 
 def validate_user_wallet_address(wallet_address, email, signed_email):
     message_hash = encode_defunct(text=email)
     signer = w3.eth.account.recover_message(message_hash, signature=signed_email)
 
-    assert wallet_address == signer
+    if not wallet_address == signer:
+        msg = _('Could not verificate wallet owner.')
+        raise exceptions.ValidationError(msg)
+
+def check_wallet_address_existence(wallet_address):
```

Comment:
this is a strange function. it checks whether wallet exists or not and raise error if exists. why?